jnigen/test/jackson_core_test/third_party/lib/com/fasterxml/jackson/core/JsonFactory.dart

```diff
@@ -920,7 +923,7 @@ class JsonFactory extends jni.JniObject {
   /// Method for constructing parser for parsing
   /// the contents of given byte array.
   ///@since 2.1
-  jsonparser_.JsonParser createParser4(jni.JniObject data) =>
+  jsonparser_.JsonParser createParser4(jni.JniArray<ffi.Int8> data) =>
```

Comment:
@dcharkes Java Byte is signed, but it might make more sense to map `byte[]` to `jniArray<ffi.Uint8>`?

Reply:
One advantage that the current situation has is that every `JPrimitive` maps to a different FFI type, I'm using this fact to for example return a `bool` for an array of `JBoolean` (`Uint8`) and return an `int` for an array of `JByte` (`Int8`).